app/route_rangers_api/utils/heatmap_data_prep.py

```diff
@@ -0,0 +1,218 @@
+# Purpose: Query database to format and prepare demographic data
+# for heatmaps
+# Author: Jimena Salinas
+# Date: Day 17th, 2024
+
+import os
+import sys
+import django
+from dotenv import load_dotenv
+import pandas as pd
+from shapely import wkt
+import geopandas as gpd
+from geopandas import GeoDataFrame
+import json
+from shapely.geometry import MultiPolygon
+
+
+def setup_django() -> None:
+    load_dotenv()
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "geodjango.settings")
+
+    # parent directory to find route_rangers_api
+    current_dir = os.path.dirname(os.path.abspath(__file__))
+    parent_dir = os.path.abspath(os.path.join(current_dir, "../../"))
+    sys.path.append(parent_dir)
+
+    django.setup()
+
+
+def fetch_demographics_data() -> pd.DataFrame:
+    from route_rangers_api.models import Demographics
+
+    demographics_queryset = Demographics.objects.all().values(
+        "census_tract",
+        "state",
+        "county",
+        "median_income",
+        "work_commute_time_15_29",
+        "work_commute_time_30_44",
+        "work_commute_time_45_59",
+        "work_commute_time_60_89",
+        "work_commute_time_less_15",
+        "work_commute_time_over_90",
+        "transportation_to_work_public",
+        "transportation_to_work_bus",
+        "transportation_to_work_subway",
+        "population",
+        "geographic_delimitation",
+    )
+    return pd.DataFrame(demographics_queryset)
```

Comment:
just as a TODO, not to address now, if we were to ingest the demographic data dynamically for the heatmaps, we might want to look into whether we could do some of this computation directly on the db to speed things up using query expressions (https://docs.djangoproject.com/en/5.0/ref/models/expressions/), maybe make a backlog ticket or TODO comment in the code.